src/IR.cpp

```diff
@@ -610,7 +610,9 @@ const char *const intrinsic_op_names[] = {
     "if_then_else",
     "if_then_else_mask",
     "image_load",
+    "image_load_texture",
     "image_store",
+    "image_store_texture",
```

Comment:
These new intrinsics are causing failures when using the C/C++ backend, as it doesn't know how to handle them and just dies. See https://buildbot.halide-lang.org/master/#/builders/35/builds/42